doc/source/overview.md

```diff
@@ -0,0 +1,32 @@
+# Overview
+
+## Raster Data
+This set of APIs provide queries for retrieving statistical and summary data from multidimensional raster datasets containing climate data.
+
+Raster datasets treat data as a regular matrix. Each dataset has a list of latitudes, a list of longitudes, a list of timestamps, and a list of variables such as temperature, precipitation, or derived values like growing degree days. A value may be accessed for every combination of latitude, longitude, timestamp, and variable, which represents the value of that variable at that particular point in time and space. 
+
+These data are interpolated from real-world observations or generated by General Circulation Models or Regional Climate Models, which are able to provide complete data coverage. Individual points may be masked off as missing, but this is typically only done to mark the boundaries of irregular geographic extents, like coastlines. The data should be assumed to be dense within the spatial and temporal boundaries.
+
+
+## Data Access
+This system is not designed to provide individual access to every point in the raster data matrix, but calculates and makes available summary data over user-defined spatial and temporal scales to support analysis and visualization. All numerical API endpoints accept a spatial area of interest (specified in URL-encoded WKT as a point or polygon) and return the mean of all cells within the designated area. If no area is designated, the mean across the entire spatial extent of the dataset is returned. 
+
+Datafiles are organized by temporal range and resolution.
+
+## Modeled Data
+
+Most data accessible by the API is climate data from a General Circulation Model (GCM) or Regional Climate Model (RCM). These numerical models represent physical processes in the atmosphere, ocean, cryosphere, and land surface of the earth. They simulate the response of the global climate system. The models output meteorological projections like temperature and precipitation. Other available variables represent statistical indices derived from the meteorological outputs.
+
+### Emissions Scenarios
+
+Each dataset has an associated emissions scenario, sometimes called `experiment`. Emissions scenarios represent a range of possible future projections for greenhouse gas concentrations. GCMs and RCMs are typically driven by historical data on greenhouse gas concentrations when modeling the past and present, before using an emissions scenario to provide input on greenhouse gas concentrations to drive future projections.
+
+### Runs
+
+Each dataset has an associated run string, sometimes called `ensemble_member`, which represents the initialization settings of the simulating model. A GCM or RCM may be run multiple times with different initialization conditions.  
```

Comment:
Similarly to "experiment," let's define terms here: A set of related runs with different initializations of the same model is called an "ensemble" (in reference to the notion of statistical ensemble).

Should we also mention (and link to more info on) the R-I-P code for this?

Reply:
I struggled writing the help with how much to try and stick  to generalities that applied to all possible data this API can serve versus how much to go into specifics on our most common case (specifically, GCM output climatologies containing a single variable of climate data). Most of our data, by far, is climatological data derived from a GCM and has an RIP code, but not all of it. So I wasn't sure whether to talk more about RIP codes or RCP codes, which apply to almost all, but not all, of the data. 

Thoughts or guidelines on generality versus detail?

Reply:
I think you are right to want to discuss the most general case. However, the term "experiment" at minimum comes from a specific context and usage, and I think that's worth explaining, otherwise it just becomes unthethered jargon. To generalize, I'd suggest you say, "we generalized from this very common case arising from GCMs to ...."

Reply:
Also, sorry about the single comments, I meant to submit a review as a single swack. I can let you know when I'm done so you don't jump every time I save a comment.